cmd/seed-controller-manager/wrappers_ee.go

```diff
@@ -48,5 +49,9 @@ func setupControllers(ctrlCtx *controllerContext) error {
 		return fmt.Errorf("failed to create GroupProjectBinding controller: %w", err)
 	}
 
+	if err := kubelbcontroller.Add(ctrlCtx.mgr, ctrlCtx.runOptions.workerCount, ctrlCtx.runOptions.workerName, ctrlCtx.runOptions.overwriteRegistry, ctrlCtx.seedGetter, ctrlCtx.clientProvider, ctrlCtx.log, ctrlCtx.versions); err != nil {
+		return fmt.Errorf("failed to create Kubelb controller: %w", err)
```

Comment:
Is it KubeLb or KubeLB?

Reply:
`KubeLB`

Reply:
`failed to create Kubelb controller` is then still wrong.

Reply:
Oops, fixed that as well.